src/DependencyInjection/Core/Community.VisualStudio.Toolkit.DependencyInjection.Core.props

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project>
+
+    <ItemGroup>
+      <Content Include="$(MSBuildThisFileDirectory)..\lib\*\*.dll" CopyToOutputDirectory="PreserveNewest" Visible="false">
```

Comment:
I am unsure what this does... copied from the other project.  @madskristensen - is this correct/needed?